src/client/components/Avatar/Avatar.css

```diff
@@ -0,0 +1,19 @@
+.avatar-images-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+.main-avatar-images {
+  max-width: 10em;
+  max-height: 10em;
+  border-bottom-left-radius: 25em;
+  border-bottom-right-radius: 25em;
+}
+
+@media only screen and (min-width: 279px) and (max-width: 1024px) {
+  .main-avatar-images {
+    max-width: 7em;
+    max-height: 7em;
+  }
+}
```

Comment:
Would be nice to add a comment to explain what screen sizes are being handled with this media query :)